packages/manager/apps/pci-load-balancer/package.json

```diff
@@ -31,9 +31,11 @@
     "@tanstack/react-query": "^5.50.1",
     "@tanstack/react-query-devtools": "^5.50.1",
     "@tanstack/react-table": "^8.17.3",
+    "axios": "^1.7.7",
     "date-fns": "^4.1.0",
     "i18next": "^23.11.5",
     "i18next-http-backend": "^2.5.2",
+    "nanoid": "^5.0.7",
```

Comment:
is it used ?

Reply:
yes, to generate a key for a select that doesn't take into account the initial value(124 bytes), but may replaced by Date.now or equivalent